experiment/autobumper/bumper/bumper.go

```diff
@@ -261,3 +536,38 @@ func generatePRBody(images map[string]string, assignment string) string {
 	testImagesSummary := generateSummary("test-image", testImageRepo, testImagePrefix, false, images)
 	return prowSummary + "\n\n" + testImagesSummary + "\n\n" + assignment + "\n"
 }
+
+func getAssignment(oncallAddress string) string {
```

Comment:
Thoughts on unit testing this and/or getOncaller?

Reply:
Added unit tests for this function.